Makefile

```diff
@@ -55,7 +56,10 @@ run-validator:
 link-development-env:
 	ln -sf .development.env .env
 
-run: build link-development-env
+link-functional-tests-env:
+	ln -sf .functional-tests.env .env
+
+run: build $(ENV_FILE_LINK_COMMAND)
 	pipenv run flask run
```

Comment:
We should make this and all the commands overrideable. As per the README to run functional test `RUNNER_ENV_FILE=.functional-tests.env make run` would currently not work. We can do something like this to make it work for this and all references of it.

```makefile
link-development-env:
	ln -sf $(RUNNER_ENV_FILE) .env

run: build link-development-env
	pipenv run flask run
```